### PR TITLE
nolog(sb) add scss/pageHeader story

### PR DESCRIPTION
**Pull request by AlexiaAlarza**
Capture d’écran 2021-07-27 à 11 14 33

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1420

File: stories/scss/page-header/page-header-basic.stories.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+				</div>
+			</div>
+			<div class="pageHeader-content-actions">
+				<label class="textfield mod-search">
+					<input class="textfield-input" type="text" placeholder="ex : Mon précieux">
+					<span class="textfield-label u-mask">Rechercher</span>
+				</label>
+				<button class="button">Button</button>
+				<button class="button mod-outline">Button</button>
+				<button class="button mod-outline mod-icon">
```
Comment (MarcFairbrother) [inline stories/scss/page-header/page-header-basic.stories.html:37]: `type="button"`

**Review by MarcFairbrother — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1420

File: stories/scss/page-header/page-header-basic.stories.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<section class="contentSection">
+	<h1>Header</h1>
+	<header class="pageHeader" [ngClass]="{'mod-withBreadcrumbs': withBreadcrumbs, 'mod-withMenu': withMenu}">
+		<nav *ngIf="withBreadcrumbs" class="breadcrumbs" aria-describedby="breadcrumbs-title">
+			<p id="breadcrumbs-title" class="u-mask">Breadcrumbs</p>
+			<ul class="breadcrumbs-list">
+				<li class="breadcrumbs-list-item"><a href="#" class="breadcrumbs-list-item-action">You</a></li>
+				<li class="breadcrumbs-list-item"><a href="#" class="breadcrumbs-list-item-action">Are</a></li>
```
Comment (shajz) [inline stories/scss/page-header/page-header-basic.stories.html:8]:
```suggestion
				<li class="breadcrumbs-list-item"><a class="breadcrumbs-list-item-action">You</a></li>
				<li class="breadcrumbs-list-item"><a class="breadcrumbs-list-item-action">Are</a></li>
```

File: stories/scss/page-header/page-header-basic.stories.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+				<button type="button" class="button mod-outline mod-icon">
+					<span aria-hidden="true" class="lucca-icon icon-ellipsis"></span>
+					<span class="u-mask">voir plus</span>
+				</button>
+			</div>
+		</div>
+		<section class="pageHeader-description">
+			<p class="u-marginBottomReset">
+				{{ pageHeaderDescription }} 
+				<a target="_blank" href="#">Lien<span aria-hidden="true" class="lucca-icon icon-outside size-smaller u-marginLeftSmallest"></span></a>.
```
Comment (shajz) [inline stories/scss/page-header/page-header-basic.stories.html:46]:
```suggestion
				<a target="_blank">Lien<span aria-hidden="true" class="lucca-icon icon-outside size-smaller u-marginLeftSmallest"></span></a>.
```

File: stories/scss/page-header/page-header-basic.stories.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+			</p>
+		</section>
+		<nav *ngIf="withMenu" class="menu">
+			<a href="" class="menu-link is-active"> tab </a>
+			<a href="" class="menu-link"> tab </a>
+			<a href="" class="menu-link"> tab </a>
+			<a href="" class="menu-link"> tab </a>
+			<a href="" class="menu-link"> tab </a>
+			<a href="" class="menu-link"> tab </a>
+			<a href="" class="menu-link"> tab </a>
```
Comment (shajz) [inline stories/scss/page-header/page-header-basic.stories.html:56]:
```suggestion
			<a class="menu-link is-active"> tab </a>
			<a class="menu-link"> tab </a>
			<a class="menu-link"> tab </a>
			<a class="menu-link"> tab </a>
			<a class="menu-link"> tab </a>
			<a class="menu-link"> tab </a>
			<a class="menu-link"> tab </a>
```

**Review by shajz — CHANGES_REQUESTED** [review]
Pareil que https://github.com/LuccaSA/lucca-front/pull/1405

**Review by shajz — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1420

**Pull request merged**